cloud-optimized-netcdf4-hdf5/index.qmd

```diff
@@ -139,7 +139,7 @@ For example, the following library defaults will impact performance and may be i
 
 * HDF5 library:
     * **Chunk cache:** The size of the HDF5's chunk cache by default is 1MB. This value is configurable. Chunks that don't fit into the chunk cache are discarded and must be re-read from the storage location each time. See also: [Improve HDF5 performance using caching](https://www.hdfgroup.org/2022/10/17/improve-hdf5-performance-using-caching/) and [h5py documentation: Chunk cache](https://docs.h5py.org/en/stable/high/file.html#chunk-cache).
-    * **Page buffer size:** The [H5Pset_page_buffer_size](https://hdfgroup.github.io/hdf5/develop/group___f_a_p_l.html#title89) and [the page_buf_size argument to h5py.File](https://docs.h5py.org/en/stable/high/file.html) should match up with the page size to optimize reading the data in pages.
+    * **Page buffer size:** The [H5Pset_page_buffer_size](https://support.hdfgroup.org/documentation/hdf5/latest/group___f_a_p_l.html#ga8008cddafa81bd1ddada23f6d9a161ca) and [the page_buf_size argument to h5py.File](https://docs.h5py.org/en/stable/high/file.html) should match up with the page size to optimize reading the data in pages.
```

Comment:
I think we want to keep https://hdfgroup.github.io/hdf5/develop/group___f_a_p_l.html#title89

The hashes to get specifically to the page_buffer anchor seem to change with every update. Also do we want Latest or Develop @abarciauskas-bgse?

Reply:
🤔 Looking at https://github.com/HDFGroup/hdf5?tab=readme-ov-file#documentation
If I hit https://support.hdfgroup.org/releases/hdf5/latest/ it takes me to https://support.hdfgroup.org/releases/hdf5/v1_14/v1_14_6/documentation/doxygen/index.html and I can search and navigate to `H5Pset_page_buffer_size` documentation at https://support.hdfgroup.org/releases/hdf5/v1_14/v1_14_6/documentation/doxygen/group___f_a_p_l.html#ga8008cddafa81bd1ddada23f6d9a161ca. I don't know a way to get a stable URL for the latest release, develop may be the best?